sway/commands/focus.c

```diff
@@ -156,6 +156,14 @@ static struct sway_node *node_get_in_direction(struct sway_container *container,
 	if (new_output) {
 		return get_node_in_output_direction(new_output, dir);
 	}
+
+	// If there is a wrap candidate, return it's focus inactive view
```

Comment:
```suggestion
	// If there is a wrap candidate, return its focus inactive view
```
Minor typo

Reply:
Thanks. Instead of applying the suggestion and adding a commit for the typo, I just fixed it locally before rebasing.